docs/stable/getting_started/docker_quickstart.md

```diff
@@ -4,92 +4,52 @@ sidebar_position: 2
+2. **ServerlessLLM CLI**: Installed on your system. You can install it using `pip install serverless-llm`.
+1. **GPUs**: At least 2 NVIDIA GPUs are necessary. If you have more GPUs, you can adjust the `docker-compose.yml` file accordingly.
+2. **NVIDIA Docker Toolkit**: This allows Docker to use NVIDIA GPUs. Follow the installation guide [here](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html).
 
-## Run ServerlessLLM using Docker
+## Run ServerlessLLM using Docker Compose
 
-First, let's start a local Docker-based ray cluster to run ServerlessLLM.
+We will use docker-compose to simplify the setup of ServerlessLLM. The `docker-compose.yml` file is located in the `examples/docker_compose/` directory of the ServerlessLLM repository.
 
-### Step 1: Build Docker Images
+### Step 1: Clone the ServerlessLLM Repository
 
-Run the following commands to build the Docker images:
+If you haven't already, clone the ServerlessLLM repository:
 
 ```bash
-docker build . -t serverlessllm/sllm-serve
-docker build -f Dockerfile.worker . -t serverlessllm/sllm-serve-worker
+git clone https://github.com/serverless-llm/serverlessllm.git
+cd serverlessllm/examples/docker_compose/
 ```
 
-### Step 2: Configuration
+### Step 2:  Configuration
 
-Ensure that you have a directory for storing your models and set the `MODEL_FOLDER` environment variable to this directory:
+Set the Model Directory
+Create a directory on your host machine where models will be stored and set the MODEL_FOLDER environment variable to point to this directory:
 
 ```bash
-export MODEL_FOLDER=path/to/models
+export MODEL_FOLDER=/path/to/your/models
 ```
 
-Also, check if the Docker network `sllm` exists and create it if it doesn't:
+Replace /path/to/your/models with the actual path where you want to store the models.
 
-```bash
-if ! docker network ls | grep -q "sllm"; then
-  echo "Docker network 'sllm' does not exist. Creating network..."
-  docker network create sllm
-else
-  echo "Docker network 'sllm' already exists."
-fi
-```
-
-### Step 3: Start the Ray Head and Worker Nodes
-
-Run the following commands to start the Ray head node and worker nodes:
-
-#### Start Ray Head Node
+### Step 3: Start the Services
 
-```bash
-docker run -d --name ray_head \
-  --runtime nvidia \
-  --network sllm \
-  -p 6379:6379 \
-  -p 8343:8343 \
-  --gpus '"device=none"' \
-  serverlessllm/sllm-serve
-```
-
-#### Start Ray Worker Nodes
+Start the ServerlessLLM services using docker-compose:
 
 ```bash
-docker run -d --name ray_worker_0 \
-  --runtime nvidia \
-  --network sllm \
-  --gpus '"device=0"' \
-  --env WORKER_ID=0 \
-  --mount type=bind,source=$MODEL_FOLDER,target=/models \
-  serverlessllm/sllm-serve-worker
-
-docker run -d --name ray_worker_1 \
-  --runtime nvidia \
-  --network sllm \
-  --gpus '"device=1"' \
-  --env WORKER_ID=1 \
-  --mount type=bind,source=$MODEL_FOLDER,target=/models \
-  serverlessllm/sllm-serve-worker
+docker-compose up -d --build
```

Comment:
Seems that in current version, it should be `docker compose` compared with `docker-compose`? ref: https://docs.docker.com/compose/releases/migrate/